packages/event-logger/plugin.schema.json

```diff
@@ -97,19 +97,42 @@
         },
```

Comment:
Had to regenerate plugin.schema.json files due to a schema change. (CI failed on uncommitted files)